src/qtism/data/content/interactions/ExtendedTextInteraction.php

```diff
@@ -313,7 +313,7 @@ public function setPatternMask($patternMask): void
      */
     public function getPatternMask(): string
     {
-        return $this->patternMask;
+        return $this->getFormat() !== TextFormat::XHTML ? $this->patternMask : '';
```

Comment:
That's not exactly what I meant. We shouldn't skip any validations by default.
QTI SDK should offer a way to optionally disable certain validations.